AK/DistinctNumeric.h

```diff
@@ -286,8 +286,8 @@ class DistinctNumeric {
     }
     constexpr Self& operator-=(Self const& other)
     {
-        static_assert(options.arithmetic, "'a+=b' is only available for DistinctNumeric types with 'Arithmetic'.");
-        this->m_value += other.m_value;
+        static_assert(options.arithmetic, "'a-=b' is only available for DistinctNumeric types with 'Arithmetic'.");
+        this->m_value -= other.m_value;
```

Comment:
This is a bit more than a typo since it's doing the wrong operation too! I wonder if this causes some breakage if there were any places that 'fixed' this with another bug? Seems odd this would not have been found sooner :thinking:

Reply:
Also you might want to add a test case in the `TestDistinctNumeric`, it seems this operator was missed out from the tests.